Mathlib/Combinatorics/Enumerative/Partition.lean

```diff
@@ -123,6 +151,21 @@ instance UniquePartitionZero : Unique (Partition 0) where
 instance UniquePartitionOne : Unique (Partition 1) where
   uniq _ := Partition.ext _ _ <| by simp
 
+lemma ofSym_one (s : Sym σ 1) : ofSym s = indiscrete 1 := by
+  ext; simp
+
+/-- The equivalence between `σ` and `1`-tuples of elements of σ -/
+def ofSym_equiv_indiscrete (σ : Type*) [DecidableEq σ] : σ ≃
+    { a : Sym σ 1 // ofSym a = indiscrete 1 } where
```

Comment:
Hmm, I'm not sure this equivalence is interesting given that it is the combination of `Sym.oneEquiv` and `alpha ~= {a : alpha // True}`

Reply:
How do you combine them? I'm having trouble with it, I can't identify `ofSym a` with `indiscrete 1`.

Reply:
I guess my problem is that I don't know how to prove `msymm_one` from https://github.com/leanprover-community/mathlib4/pull/12572 without this lemma.

Reply:
I'll have a look tomorrow evening. Please remind me if I forget

Reply:
I made a commit on the other branch removing the lemma. I don't quite like the proof, but it works. I can either revert that commit or remove the lemma here, let me know which one is preferred.

Reply:
Ok cool!